docs/helm/README.md

```diff
@@ -184,6 +184,16 @@ repository:
     org.alfresco.repo.security.authentication.identityservice: debug
 ```
 
+In addition with the Log4j we have the logging level with Log4j2, below is the example:
+
+```yaml
+repository:
+  ...
+  extraLogStatements:
+    logger.alfresco-repo-content-transform-TransformerDebug.name: org.alfresco.repo.content.transform.TransformerDebug
+    logger.alfresco-repo-content-transform-TransformerDebug.level: debug
+```
+
```

Comment:
I would place the log4v2 example first as it's now the default.
Also remove the identityservice debug line so it's more visible that with log4v2 2 lines are required while one only for log4jv1